db/seeds.d/107-enable_dynflow.rb

```diff
@@ -4,5 +4,5 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 User.as(::User.anonymous_api_admin.login) do
-  Setting.find_by(:name => "dynflow_enable_console").update!(:value => true) if Rails.env.development?
+  Setting.find_by(:name => "dynflow_enable_console")&.update!(:value => true) if Rails.env.development?
```

Comment:
When would this setting not exist?

Reply:
@lfu I have no idea @jeremylenz told me to do this since it's used in other places and blocking the katello-devel boxes

Reply:
I don't even know what `&` does lol

Reply:
> When would this setting not exist?

Apparently it doesn't exist when building a new dev box :shrug:

Reply:
I noticed the following behavior on my dev box
```ruby
[1] pry(main)> Setting['dynflow_enable_console']
=> true
[2] pry(main)> Setting.find_by(:name => "dynflow_enable_console")
=> nil
```

looking at https://github.com/theforeman/foreman/blob/develop/app/models/setting.rb#L83-L90 foreman is using a different mechanism.  My guess is `Foreman.settings` is yet to be run when the seed script gets run. So the suggested fix tries to do `Setting["dynflow_enable_console"] = true` if its not already set.